ui/src/components/VoxettaUserForm.js

```diff
@@ -46,7 +46,7 @@ export class VoxettaUserForm extends LitElement {
                 flex-wrap: wrap;
                 justify-content: center; 
                 text-align: center;  
-                width: 400px; 
+                width: 100vw; 
```

Comment:
set width here, not in constructor. Does it not take hold here?

Reply:
apologies forgot to remove from the constructor after I was done experimenting with width--removed from constructor